src/hooks/useDailyStatistics/utils.ts

```diff
@@ -1,7 +1,9 @@
 import { chain, sumBy } from "lodash";
-import { DailyStatistics, CampaignPolicy } from "../../types";
 import { formatQuantityText } from "../../components/CustomerQuota/utils";
+import { DailyStatistics, CampaignPolicy } from "../../types";
+
```

Comment:
Not sure why there's this change, when the import is already there

Reply:
rearranged, removed line 4 spacing